test/functional/page_objects/settings_page.js

```diff
@@ -154,44 +144,39 @@ export function SettingsPageProvider({ getService, getPageObjects }) {
       return await testSubjects.find('indexPatternTitle');
     }
 
-    getConfigureHeader() {
-      return remote.setFindTimeout(defaultFindTimeout)
-        .findByCssSelector('h1');
+    async getConfigureHeader() {
+      return await find.allByCssSelector('h1');
     }
 
-    getTableHeader() {
-      return remote.setFindTimeout(defaultFindTimeout)
-        .findAllByCssSelector('table.euiTable thead tr th');
+    async getTableHeader() {
+      return await find.allByCssSelector('table.euiTable thead tr th');
     }
 
-    sortBy(columnName) {
-      return remote.setFindTimeout(defaultFindTimeout)
-        .findAllByCssSelector('table.euiTable thead tr th button')
-        .then(function (chartTypes) {
-          function getChartType(chart) {
-            return chart.getVisibleText()
-              .then(function (chartString) {
-                if (chartString === columnName) {
-                  return chart.click()
-                    .then(function () {
-                      return PageObjects.header.waitUntilLoadingHasFinished();
-                    });
-                }
-              });
+    async sortBy(columnName) {
+      const chartTypes = find.allByCssSelector('table.euiTable thead tr th button');
+      function getChartType(chart) {
+        return chart.getVisibleText().then(function (chartString) {
+          if (chartString === columnName) {
+            return chart.click().then(function () {
+              return PageObjects.header.waitUntilLoadingHasFinished();
+            });
```

Comment:
Love the refactoring done here.  Could make even more readable, imo, with:

```js
    async sortBy(columnName) {
      const chartTypes = find.allByCssSelector('table.euiTable thead tr th button');
      function async getChartType(chart) {
        const chartString = await chart.getVisibleText();
        if (chartString === columnName) {
           await chart.click();
           await PageObjects.header.waitUntilLoadingHasFinished();
        };
      }

      const getChartTypesPromises = chartTypes.map(getChartType);
      return Promise.all(getChartTypesPromises);
    }
```

take it or leave, your current modifications are already an improvement.